components/gainsight_nxt/gainsight_nxt.app.mjs

```diff
@@ -1,11 +1,92 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "gainsight_nxt",
-  propDefinitions: {},
+  propDefinitions: {
+    objectName: {
+      type: "string",
+      label: "Custom Object",
+      description: "The name of the custom object.",
+      async options() {
+        const { data } = await this.listCustomObjects();
+        return data/*?.filter?.((obj) => obj.objectType === "CUSTOM")*/.map(( {
```

Comment:
Is the filter meant to be commented out?

Reply:
It is, I commented it out for testing and forgot to remove the block afterwards - doing it now, thanks.

For the record, it seems several custom object types are not actually classified explicitly as "custom", hence why I removed this filter (it was returning no objects whatsoever with it)